lib/helper.js

```diff
@@ -394,20 +393,20 @@ PokerHelper.prototype.getMyPos = function getMyPos(lineup, address) {
  * @param
  * @returns
  */
-PokerHelper.prototype.findMinRaiseAmount = function findMinRaiseAmount(lineup, dealer, lastRoundMaxBet, state) {
+PokerHelper.prototype.getMinRaiseAmount = function getMinRaiseAmount(lineup, state, dealer, lastRoundMaxBet, bbAmount) {
   if (!lineup || dealer === undefined) {
     throw new Error('invalid params.');
   }
   // TODO: pass correct state
-  const max = this.getMaxBet(lineup, state);
+  const bbPos = this.getBbPos(lineup, dealer, state);
+  const max = this.getMaxFromActive(lineup, state, false, dealer, bbAmount, bbPos);
```

Comment:
this is taking `activeLineup`, but the public interface should not receive `activeLineup`, or at least there should be a check.